Example/Pods/Local Podspecs/Artsy+UIFonts.podspec.json

```diff
@@ -18,9 +18,10 @@
   },
   "source_files": "Pod/Classes",
   "resources": "Pod/Assets/*",
+  "preserve_paths": "Pod/Scripts/*",
   "frameworks": [
     "UIKit",
     "CoreText"
   ],
-  "prepare_command": "echo \"HELLO\"\necho $PWD\necho $ENV"
+  "prepare_command": "./Pod/Scripts/ArtsySetup.rb"
```

Comment:
Was this previous command important for anything?

Reply:
I didn't change this myself so I'm not totally sure why it changed or what the old one is for.

Reply:
Oh @alloy - this podspec was old. When I `pod update`d, it changed to match the current one on `master`.